packages/ui/src/printer.ts

```diff
@@ -0,0 +1,10 @@
+export function printer(content: string): void {
```

Comment:
same here, `local-content-printer.ts`.

you can consider putting both the localContentDownloader and localContentPrinter under same file `local-content-utils.ts`